bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -23,7 +23,7 @@ metadata:
     description: Provides tracing, monitoring and troubleshooting for microservices-based
       distributed systems
     operators.openshift.io/infrastructure-features: '["disconnected"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0+git
```

Comment:
should this be 1.16 or 1.17? 🤔

Reply:
@rubenvp8510 I'm not sure.  This code is all generated, right?